lua/solarized/theme.lua

```diff
@@ -1,460 +1,471 @@
+        SpecialComment      = ts['@comment.documentation'], -- special things inside a comment
+        Debug               = ts['@debug'],                 -- debugging statements
+        Underlined          = ts['@text.underline'],        -- text that stands out, HTML links
+        Error               = ts['@error'],                 -- any erroneous construct
+        Todo                = ts['@text.todo'],             -- anything that needs extra attention; mostly the keywords TODO FIXME and XXX
+
+        htmlLink            = { fg = solarized.link, style = 'underline' },
+        htmlH1              = { fg = solarized.cyan, style = 'bold' },
+        htmlH2              = { fg = solarized.red, style = 'bold' },
+        htmlH3              = { fg = solarized.green, style = 'bold' },
+        htmlH4              = { fg = solarized.purple, style = 'bold' },
+        htmlH5              = { fg = solarized.yellow, style = 'bold' },
+        markdownH1          = { fg = solarized.cyan, style = 'bold' },
+        markdownH2          = { fg = solarized.red, style = 'bold' },
+        markdownH3          = { fg = solarized.green, style = 'bold' },
         markdownH1Delimiter = { fg = solarized.cyan },
         markdownH2Delimiter = { fg = solarized.red },
         markdownH3Delimiter = { fg = solarized.green },
-	}
-
-	-- Options:
-
-	-- Italic comments
-	if vim.g.solarized_italic_comments == true then
-		syntax.Comment =		{fg = solarized.comments, bg = solarized.none, style = 'italic'} -- italic comments
-	else
-		syntax.Comment =		{fg = solarized.comments} -- normal comments
-	end
-
-	-- Italic Keywords
-	if vim.g.solarized_italic_keywords == true then
-		syntax.Conditional =		{fg = solarized.yellow, bg = solarized.none, style = 'italic'} -- italic if, then, else, endif, switch, etc.
-		syntax.Keyword =			{fg = solarized.yellow, bg = solarized.none, style = 'italic'} -- italic for, do, while, etc.
-		syntax.Repeat =				{fg = solarized.yellow, bg = solarized.none, style = 'italic'} -- italic any other keyword
-	else
-		syntax.Conditional =		{fg = solarized.yellow} -- normal if, then, else, endif, switch, etc.
-		syntax.Keyword =			{fg = solarized.yellow} -- normal for, do, while, etc.
-		syntax.Repeat =				{fg = solarized.yellow} -- normal any other keyword
-	end
-
-	-- Italic Function names
-	if vim.g.solarized_italic_functions == true then
-		syntax.Function =		{fg = solarized.blue, bg = solarized.none, style = 'italic'} -- italic funtion names
-	else
-		syntax.Function =		{fg = solarized.blue} -- normal function names
-	end
-
-	if vim.g.solarized_italic_variables == true then
-		syntax.Identifier =				{fg = solarized.gray, bg = solarized.none, style = 'italic'}; -- any variable name
+    }
+
+    -- Options:
+
+    -- Italic comments
+    if vim.g.solarized_italic_comments == true then
+        syntax.Comment = { fg = solarized.comments, bg = solarized.none, style = 'italic' } -- italic comments
```

Comment:
I've not bothered with the italic feature, I found it an odd feature, I can propose something if you want to move forward with this PR.